lib/msal-browser/src/interaction_client/PopupClient.ts

```diff
@@ -833,4 +871,22 @@ export class PopupClient extends StandardInteractionClient {
         const homeAccountId = request.account && request.account.homeAccountId;
         return `${BrowserConstants.POPUP_NAME_PREFIX}.${this.config.auth.clientId}.${homeAccountId}.${this.correlationId}`;
     }
+
+    /** 
+     * Extracts the server response from the popup window
+     */
+    extractServerResponseStringFromPopup(
+        popupWindow: Window,
+        href: string
+    ): string {
+        let serverResponseString;
+        if(this.config.auth.protocolMode === ProtocolMode.OIDC && 
+            this.config.auth.OIDCOptions?.serverResponseType === ServerResponseType.QUERY) {
```

Comment:
Now that the validation is done in the config builder and you've given this a default value we can just directly check responseType

```suggestion
        if(this.config.auth.OIDCOptions.serverResponseType === ServerResponseType.QUERY) {
```